src/pages/SignupHandler/index.js

```diff
@@ -119,13 +133,45 @@ const SignupHandler = props => {
     }
   };
 
-  const handleSignupSubmit = () => {
+  const handleSignupSubmit = async () => {
     const { dispatch } = props;
-    dispatch(routerRedux.push('/login'));
+    const response = await dispatch({
+      type: 'auth/registerUser',
+      payload: {
+        firstName,
+        lastName,
+        username: firstName.toLowerCase(),
+        password,
+        email,
+      },
+    });
+    const { status, message } = response;
+    setAlertComponent({
+      message,
+      variant: status === 'failure' ? 'danger' : 'success',
+    });
+    setAlertOpen(true);
+    if (status === 'success') {
+      // wait for a couple
+      // of seconds before taking the
+      // user to login page.
```

Comment:
Funny, I somehow hadn't noticed the duplication of the comment text; so the login comment was copied from here, where you really _are_ redirecting to the login page, after registration. Ah.

Would it make sense to consolidate the alert and wait into a common method somewhere, though, so the behavior remains consistent... even though the timeout action differs? (It'd be easy enough to pass a one-line lambda in each case, for example.)